clang/lib/Driver/ToolChains/SYCL.h

```diff
@@ -1,4 +1,5 @@
-//===--- SYCL.h - SYCL ToolChain Implementations -----------------*- C++ -*-===//
+//===--- SYCL.h - SYCL ToolChain Implementations -----------------*- C++
+//-*-===//
```

Comment:
```suggestion
//===--- SYCL.h - SYCL ToolChain Implementations ----------------*- C++ -*-===//
```
Would be more fitting to adjust the line length instead of adding a new line.